Makefile

```diff
@@ -307,6 +307,17 @@ remove_lib_wrapped:
 remove_locales_wrapped:
 	for f in $(LANGUAGES); do $(RM) $(DESTDIR)$(LOCALEDIR)/$$f/LC_MESSAGES/$(PACKAGE).mo; done
 
+test:
```

Comment:
I think we want this target called `check`, and it should exit non-zero on any test failures.

And we should run it from GitHub Actions, but that may be a separate PR or at least separate commit.